features/form_settings_tests.feature

```diff
@@ -80,12 +80,12 @@ Scenario: Test the 'Change Language' setting
 	Then I select form "Question Types"
 
 	Then I select "Change Language" menu item
-	Then I see a list that contains all of these items "en,hin"
-	Then I touch the "hin" text
+	Then I see a list that contains all of these items "English,हिंदी"
```

Comment:
I've also made these changes here https://github.com/dimagi/commcare-ui-tests/pull/57/files which is the branch targeted by AWS